forte/mrdsrg-spin-integrated/dsrg_mrpt3.cc

```diff
@@ -273,6 +273,13 @@ void DSRG_MRPT3::build_tei(BlockedTensor& V) {
     }
 }
 
+SharedMatrix DSRG_MRPT3::compute_gradient() {   
```

Comment:
There is another way to do this: implement and make `compute_gradient()` throw an exception in the base class, and then overload it only for the classes for which it is implemented. This extra code in each of the derived classes is difficult to maintain.